### ISSUE TITLE
Thank you very much for providing UNET code. The bce_dice_coefficients loss function mentioned above. But I don't understand how it can solve the problem of category imbalance.

### ISSUE THREAD

**Issue by huang229** [jis478/Tensorflow#2]
Thank you very much for providing UNET code. The bce_dice_coefficients loss function mentioned above. But I don't understand how it can solve the problem of category imbalance.
